--- libbb/hash_md5_sha.c	2024-09-26 08:00:52.000000000 -0600
+++ libbb/hash_md5_sha.c	2025-05-14 17:19:31.321906938 -0600
@@ -1311,11 +1311,8 @@
 	common64_end(ctx, /*swap_needed:*/ BB_LITTLE_ENDIAN);
 
 	hash_size = 8;
-	if (ctx->process_block == sha1_process_block64
-#if ENABLE_SHA1_HWACCEL
-	 || ctx->process_block == sha1_process_block64_shaNI
-#endif
-	) {
+	if (ctx->process_block == sha1_process_block64)
+	{
 		hash_size = 5;
 	}
 	/* This way we do not impose alignment constraints on resbuf: */
